README.md

```diff
@@ -1,4 +1,4 @@
-![PODMAN logo](https://cdn.rawgit.com/kubernetes-incubator/cri-o/master/logo/crio-logo.svg)
+![PODMAN logo](https://cdn.rawgit.com/kubernetes-incubator/cri-o/master/logo/podman-logo.png)
```

Comment:
Probably want to update the repo path here to not use cri-o.  Based on our current `play.png` examples, I think we can use relative links.